content/instructor/quick-resources/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-04-13
+title: 'Quick Resources Page'
+description: "Getting ready to record? Find the information you need on one page!"
+categories: ['instructor']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_screencasting.png'
+redirect_from:
+
+---
+This **Quick Resources Page** compiles technical specifications that you might need to reference as you begin recording your lessons!
+
+⚠️ First, check out this comprehensive course by John to get started: [Record Badass Screencasts for egghead.io](https://egghead.io/courses/record-badass-screencasts-for-egghead-io)
+
+|     |     |      |
+|:-----:|:-----:|:------:|
+|[Desktop Environment](#desktop-environment)|[Audio Environment](#audio-environment)|[Recording](#recording)|
```

Comment:
This is a markdown table as of right now, could use some styling?
![Screen Shot 2020-04-23 at 3 17 38 PM](https://user-images.githubusercontent.com/6188161/80140226-a5f22880-8575-11ea-9bf4-189fafa421d4.png)